docker/images/pinot/etc/jmx_prometheus_javaagent/configs/controller.yml

```diff
@@ -204,6 +204,18 @@ rules:
   cache: true
   labels:
     version: "$2"
+- pattern: "\"org.apache.pinot.common.metrics\"<type=\"ControllerMetrics\", name=\"pinot.controller.tableConsumptionPaused.([^\\.]*?)_(OFFLINE|REALTIME)\"><>(\\w+)"
+  name: "pinot_controller_tableConsumptionPaused_$3"
+  cache: true
+  labels:
+    tableName: "$1"
+    tableType: "$2"
```

Comment:
nit: I think table consumption only applies to realtime table?

Reply:
Maybe in future we can add an ability to pause consumption in offline tables too? I think we can keep this label.